Source/Core/Core/IOS/Network/KD/NWC24Config.cpp

```diff
@@ -216,4 +216,16 @@ void NWC24Config::SetEmail(const char* email)
   strncpy(m_data.email, email, MAX_EMAIL_LENGTH);
   m_data.email[MAX_EMAIL_LENGTH - 1] = '\0';
 }
+
+std::string_view NWC24Config::GetMlchkid() const
+{
+  const size_t size = strnlen(m_data.mlchkid, MAX_MLCHKID_LENGTH);
+  return {m_data.mlchkid, size};
+}
+
+std::string NWC24Config::GetCheckURL() const
```

Comment:
Any particular reason this one is a `string` and the other is a `string_view`?

Reply:
The `HttpRequest::Post` function takes a `const std::string&` rather than a `string_view` which is why I did that